packages/website/src/theme.js

```diff
@@ -13,13 +21,15 @@ export default {
   grayDark: "#333",
 
   Button: css`
+    ${Button};
```

Comment:
Why not `defaultTheme.Button`? We're already importing the default `defaultTheme`. I don't think we need to import separate members.